docs/1.0. Certificate Provisioning.md

```diff
@@ -288,9 +300,11 @@ An EST Client SHOULD periodically check the revocation status of both the Root C
 
 ## Notes to Implementers
 
+- The Random Number Generator (RNG) used to generate keys for each digital signature suite SHOULD have sufficient entropy to generate keys with the required length
```

Comment:
As discussed, this could probably be expanded to recommend that both sufficient entropy is available to avoid delays, and to ensure that when deterministic RNGs are used they don't result in keys which are ineffective.